usng.js

```diff
@@ -281,19 +355,19 @@
          utmcoords[2] = zone
 
          ***************************************************************************/
-        LLtoUTM: function (lat,lon,utmcoords,zone) {
+        LLtoUTM: function (lat,lon,utmcoords = [],zone) {
             // utmcoords is a 2-D array declared by the calling routine
             // note: input of lon = 180 or -180 with zone 60 not allowed; use 179.9999
 
             lat = parseFloat(lat);
             lon = parseFloat(lon);
 
-        // Constrain reporting USNG coords to the latitude range [80S .. 84N]
-        /////////////////
-            if (lat > 84.0 || lat < -80.0){
-                return this.UNDEFINED_STR;
-            }
-        //////////////////////
+            // Constrain reporting USNG coords to the latitude range [80S .. 84N]  
```

Comment:
✏️ As these are just whitespace changes, unless they affirmatively match a new standard formatter, they should be put back the way they were.